e3nn/o3/rotation.py

```diff
@@ -396,9 +420,9 @@ def angles_to_quaternion(alpha, beta, gamma):
         matrices of shape :math:`(..., 4)`
     """
     alpha, beta, gamma = torch.broadcast_tensors(alpha, beta, gamma)
-    qa = axis_angle_to_quaternion(torch.tensor([0, 0, 1.0]), alpha)
-    qb = axis_angle_to_quaternion(torch.tensor([0, 1, 0.0]), beta)
-    qc = axis_angle_to_quaternion(torch.tensor([0, 0, 1.0]), gamma)
+    qa = axis_angle_to_quaternion(torch.tensor([0, 1, 0.0]), alpha)
```

Comment:
I recommend consistent float literals like 1.0 or 1. instead of int literals like 1 (no period). It can be helpful to readers who may not expect transparent casting.